### ISSUE TITLE
graphical model for sequence collections

### ISSUE THREAD

**Issue by RNAer** [biocore/scikit-bio#1248]
This came up in the code review yesterday (@tanaes). It would be great to relate the sequences into a coherent entity in order to represent contigs in one genome or pan-genomes. We argue that it should be directed, cyclic (for circular genomes) or acyclic (for linear genomes) graphic model with distance between the nodes (contigs) as the gaps between contigs.

The advantages are 1) bring contigs into synteny and order and 2) represent the genomic diversity between strains as branches in the graph.

This shouldn't be difficult to implement with `networkx`?

**Comment by mortonjt** [comment]
:+1:  There seems to be quite a few benefits supporting formats such as
[FASTG](http://fastg.sourceforge.net/) as mentioned by @tanaes yesterday.

**Comment by gregcaporaso** [comment]
I agree that this could be a useful addition. We'd need to have it take lower priority than some of the other objects if a new object were required, but an RFC that started to explore some possibilities for how this would be represented might make sense. It's possible that we'd just need a parser that can load sequences as nodes in a networkx graph, which is pretty straight-forward to do.

**Comment by mortonjt** [comment]
CC @fedarko since he has already developed a parser for his metagenome assembly visualization.
